doc/connectivity/bluetooth/api/mesh/sar_cfg.rst

```diff
@@ -0,0 +1,243 @@
+.. _bluetooth_mesh_sar_cfg:
+
+Segmentation and reassembly (SAR)
+#################################
+
+Segmentation and reassembly provides a way of handling larger upper transport layer messages in a
+mesh network, with a purpose of enhancing the Bluetooth mesh throughput. The segmentation and
+reassembly mechanism is used by the lower transport layer.
+
+The lower transport layer takes Upper Transport PDUs from the upper transport layer and sends them to the lower transport layer on a peer device.
+If the Upper Transport PDU fits, it is sent in a single Lower Transport PDU.
+For longer packets, which do not fit into a single Lower Transport PDU, the lower transport layer performs segmentation, splitting the Upper Transport
+PDU into multiple segments.
+
+The receiving lower transport layer on the other device reassembles the segments into a single Upper Transport PDU before passing it
+up the stack. Delivery of a segmented message is acknowledged by the lower transport layer of the receiving node.
+
+Delivery of an unsegmented message is not acknowledged.
+But, an Upper Transport PDU that fits into one Lower Transport PDU can also be sent as a single-segment segmented message when acknowledgment
+by the lower transport layer is required.
+Set the ``send rel`` flag (see :c:struct:`bt_mesh_msg_ctx`) to use the reliable message transmission and acknowledge single-segment segmented messages.
+
+The transport layer is able to transport up to 32 segments with its SAR mechanism, with a maximum message (PDU) size of 384 octets.
+To configure number of segments and the message size for the Bluetooth mesh stack, use the following Kconfig options:
+
+* :kconfig:option:`CONFIG_BT_MESH_SEG_BUFS` to set the number of segment buffers.
+* :kconfig:option:`CONFIG_BT_MESH_RX_SEG_MAX` to set the maximum number of segments in an incoming message.
+* :kconfig:option:`CONFIG_BT_MESH_TX_SEG_MAX` to set the maximum number of segments in an outcoming message.
+
+Never more than one segmented message is transmitted at the time, but the Bluetooth mesh stack allows you to define buffers that will queue a defined
```

Comment:
Maybe something like this instead: "Only one segmented message is transmitted at a time, but the Bluetooth mesh stack allows you to **configure** buffers that can queue a specified number of incoming or outgoing segmented messages."